documentation/en/api-v2-unstable-methods.md

```diff
@@ -0,0 +1,56 @@
+# Groups
+* [Chain](#Chain)
+  * [ChainGetTipSet](#ChainGetTipSet)
+## Chain
+The Chain method group contains methods for interacting with
+the blockchain.
+
+
+### ChainGetTipSet
+ChainGetTipSet retrieves a tipset that corresponds to the specified selector
+criteria. The criteria can be provided in the form of a tipset key, a
+blockchain height including an optional fallback to previous non-null tipset,
+or a designated tag such as "latest" or "finalized".
+
+The "Finalized" tag returns the tipset that is considered finalized based on
+the consensus protocol of the current node, either Filecoin EC Finality or
+Filecoin Fast Finality (F3). The finalized tipset selection gracefully falls
+back to EC finality in cases where F3 isn't ready or not running.
+
+In a case where no selector is provided, an error is returned. The selector
+must be explicitly specified.
+
+For more details, refer to the types.TipSetSelector and
+types.NewTipSetSelector.
```

Comment:
Is someone looking at this in the future at https://docs.filecoin.io/reference/json-rpc/chain or https://github.com/filecoin-project/lotus/blob/master/api/README.md going to know how to navigate to `chain/types/tipset_selector.go` to find this info?

For example, I assume if in the 2025-04-09 meeting with Zondax if we just show this Markdown that they don't have enough context to know about the flexibility of TipSetSelector.

Reply:
heh, yeah that README probably should be nuked, I didn't even realise it existed

Reply:
https://github.com/filecoin-project/lotus/pull/13017

Reply:
We spoke about this during 2025-04-08 standup.  It's really unfortunate that our docsgen process is not better for consumers, particularly consumers of other languages besides Go.

For this particular issue at hand of "how does someone learn about the rich `TipSetSelector` type"?
* They won't be able to get that from our generated `documentation/en/api-v2-unstable-methods.md`.
* A Go developer can learn about this type if they look at the godocs
* Maybe the thing to do is to create a side document (Notion, hackmd) where we can state the API contract and richly format it while we're trying to get feedback.  (It's a later bigger item to get Lotus APIs and docs to be contract driven.)

Reply:
If it helps, I created https://www.notion.so/filecoindev/Lotus-F3-aware-APIs-1cfdc41950c180ae97fef580e79427d5 for housing any API docs we want to have outside the repo while we're in "actively getting user feedback mode".

Anyways, I think this conversation can be resolved as there's nothing more to be actioned here (but I left it open in case we want to make clear that this problem still exists and will need large followup work at some later point to correct.()

Reply:
I have created a sub-page in notion under Lotus F3 Aware APIs to document the v2 API design/implementation so far: https://www.notion.so/filecoindev/Filecoin-V2-APIs-1d0dc41950c1808b914de5966d501658

It is not finished yet but should help overcome the shortcomings of the generated docs in the meantime.